generator/cmd/generateall.ts

```diff
@@ -91,7 +91,10 @@ executeSynchronous(async () => {
                     const readme = validateAndReturnReadmePath(localPath, autoGenConfig.readmeFile || autoGenConfig.basePath);
                     pkg.packageName = getPackageString(readme);
 
+                    const startTime = Date.now();
                     const newConfigs = await generateSchemas(readme, autoGenConfig);
+                    const generationTime = Date.now() - startTime;
+                    console.log(`Time taken to generate ${autoGenConfig.basePath} : ${generationTime} ms.`);
```

Comment:
To differentiate the message, consider adding colors, like here: https://github.com/Azure/azure-resource-manager-schemas/blob/3134761b9f051d366342f4f9fcc0b36fc03f6259/generator/cmd/listbasepaths.ts#L24